packager/third_party/mbedtls/CMakeLists.txt

```diff
@@ -14,4 +14,5 @@ set(ENABLE_TESTING OFF)
 set(GEN_FILES OFF)
 
 # With these set in scope of this folder, load the library's own CMakeLists.txt.
+add_compile_options(-Wno-unused-but-set-variable)
```

Comment:
This is causing a build error on Windows:

`cl : command line  error D8021: invalid numeric argument '/Wno-unused-but-set-variable' [D:\a\shaka-packager\shaka-packager\build\packager\third_party\mbedtls\source\library\mbedcrypto.vcxproj]`

For similar situations in the past, we have had to use `if(NOT MSVC)` ... `endif()` to make the flag conditional on the compiler.